trust/list.c

```diff
@@ -158,7 +159,8 @@ list_iterate (p11_enumerate *ex,
 				p11_buffer_init (&buf, 1024);
 				bytes = attr->pValue;
 				p11_url_encode (bytes, bytes + attr->ulValueLen, "", &buf);
-				printf ("    public-key-info: %.*s\n", (int)buf.len, (char *)buf.data);
+				p11_print_value (stdout, "public-key-info:", NULL, 4, 1, P11_PRINT_BOLD);
+				printf (" %.*s\n", (int)buf.len, (char *)buf.data);
```

Comment:
For this, I would rather use varargs, calling `vfprintf` internally: `p11_printf_value (stdout, "public-key-info", 4, P11_PRINT_BOLD, "%.*s", buf.len, buf.data)`.